src/vl-upload.js

```diff
@@ -147,8 +158,10 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
             <button type="button" class="vl-upload__element__button vl-link">
               <i class="vl-vi vl-vi-paperclip" aria-hidden="true"></i>
               <span class="vl-upload__element__button__container"></span>
+              <slot id="title-slot" name="title"></slot>
             </button>
             <small></small>
+            <slot id="sub-title-slot" name="sub-title"></slot>
```

Comment:
is subtitle doorgaans niet gewoon aan elkaar vast

Reply:
ja ik had me gebaseerd op de documentatie in de webcomponent waar ze nu als attribuut ...-sub-title gebruiken en verder  niet meer bij stilgestaan. Maar idd het moet eigenlijk subtitle zijn. Wat doe ik, pas ik het aan voor de slots?

Reply:
Als het al gebruikt wordt als `sub-title` zou ik het zo laten :)

Reply:
ja denk dat consistentie ook wel beter is nu miss eentje om in de backlog te stoppen als aanpassing